_translator-files/po/de/Installation-for-Windows.po

```diff
@@ -167,10 +165,9 @@ msgstr "Je nach Hardware musst du ein bisschen herumprobieren. Wenn alles funkti
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:50
-#, fuzzy, no-wrap
-#| msgid "**Tip:** Set up your sound card while you're [connected to a server](Getting-Started#connecting-to-a-server-and-testing-your-sound) to hear your instrument or voice and check if everything is correctly set up; but first read on.\n"
+#, no-wrap
 msgid "**Tip:** Set up your sound card while you're [connected to a Server](Getting-Started#connecting-to-a-server-and-testing-your-sound) to hear your instrument or voice and check if everything is correctly set up; but first read on.\n"
-msgstr "**Tipp:** Richte deine Sondkarte ein, während du [mit einem Server verbunden bist](Getting-Started#mit-einem-server-verbinden-und-audio-testen), damit du dein Instrument oder deine Stimme hören kannst. Dadurch kannst du prüfen, ob alles richtig eingerichtet ist, aber lies ersteinmal weiter.\n"
+msgstr "**Tipp:** Richte deine Sondkarte ein, während du [mit einem Server verbunden bist](Getting-Started#connecting-to-a-server-and-testing-your-sound), damit du dein Instrument oder deine Stimme hören kannst. Dadurch kannst du prüfen, ob alles richtig eingerichtet ist, aber lies erst einmal weiter.\n"
```

Comment:
The part after the # should be translated - matching exactly the paragraph title it refers to.

Reply:
Linked to corresponding paragraph in "Getting-Started.po"